src/test/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerConfirmServiceTest.java

```diff
@@ -171,6 +175,7 @@ public void shouldNotThrowValidationErrorWhenProcessedByProcessServerButDocument
         final CaseData caseData = caseData();
         caseData.getApplication().setSolSignStatementOfTruth(YES);
         caseData.getApplication().setServiceMethod(SOLICITOR_SERVICE);
+        caseData.setApplicationType(ApplicationType.SOLE_APPLICATION);
```

Comment:
Can we add a test to confirm the state doesn't move when the Aos date is set?

Reply:
Added to SetConfirmServiceStateTest after refactor.